releasenotes/notes/process-agent-add-batch-size-env-vars-df7721763f008554.yaml

```diff
@@ -0,0 +1,4 @@
+enhancements:
+  - |
+    Add DD_PROCESS_CONFIG_MAX_PER_MESSAGE env variable to set process_config.max_per_message
```

Comment:
```suggestion
    Add the ``DD_PROCESS_CONFIG_MAX_PER_MESSAGE`` env variable to set the  ``process_config.max_per_message``.
```

Reply:
Thanks for the feedback, Kaylyn. I've just updated the PR with it!